src/rules/require-optimized-style-attribute.ts

```diff
@@ -32,7 +32,7 @@ export default createRule("require-optimized-style-attribute", {
         })
       },
       SvelteAttribute(node) {
-        if (node.key.name !== "style") {
+        if (node.key.name !== "style" || !node.value.length) {
```

Comment:
Can `node.value` be `null`? `<span style>`

Reply:
```suggestion
        if (node.key.name !== "style" || !node.value?.length) {
```

Reply:
> Can node.value be null? `<span style>`

Currently it can not be `null`, but it is different from JSX AST, so it may be a bug of AST generated by svelte-eslint-parser 🤔